Allow for calling lift_fresh_copy manually

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #113923

In this case, the input could be fake!  Just treat it normally in that case.

Fixes https://github.com/pytorch/pytorch/issues/113331

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @avikchaudhuri @gmagogsfm @zhxchen17 @tugsbayasgalan @angelayi @suo @ydwu4

ezyang committed:
Allow for calling lift_fresh_copy manually

In this case, the input could be fake!  Just treat it normally in that case.

Fixes https://github.com/pytorch/pytorch/issues/113331

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

[ghstack-poisoned]

ezyang committed:
Update on "Allow for calling lift_fresh_copy manually"


In this case, the input could be fake!  Just treat it normally in that case.

Fixes https://github.com/pytorch/pytorch/issues/113331

Signed-off-by: Edward Z. Yang <ezyangmeta.com>

cc avikchaudhuri gmagogsfm zhxchen17 tugsbayasgalan angelayi suo ydwu4

[ghstack-poisoned]

ezyang commented:
@pytorchbot merge

leslie-fang-intel commented on test/export/test_export.py:
```diff
@@ -1548,6 +1548,34 @@ def forward(self, x):
+
+        m = M().eval()
+        # res_ref = m(tensor_cpu, mask_cpu)
+        # print("res_ref is: {}".format(res_ref), flush=True)
```
Maybe delete these comments? Sorry, I left it in the example code in the issue.

ezyang replied:
I thought they were good comments 📦

ezyang commented:
@pytorchbot merge

ezyang committed:
Update on "Allow for calling lift_fresh_copy manually"


In this case, the input could be fake!  Just treat it normally in that case.

Fixes https://github.com/pytorch/pytorch/issues/113331

Signed-off-by: Edward Z. Yang <ezyangmeta.com>

cc avikchaudhuri gmagogsfm zhxchen17 tugsbayasgalan angelayi suo ydwu4

[ghstack-poisoned]

ezyang commented:
@pytorchbot merge
